test/integration/ae_mdw_web/controllers/activity_controller_test.exs

```diff
@@ -75,5 +75,51 @@ defmodule Integration.AeMdwWeb.ActivityControllerTest do
                |> get("/v2/accounts/#{invalid_account}/activities")
                |> json_response(400)
     end
+
+    test "it displays int transfers", %{conn: conn} do
+      account = "ak_dArxCkAsk1mZB1L9CX3cdz1GDN4hN84L3Q8dMLHN4v8cU85TF"
+      height = 665_679
+
+      assert %{"data" => [first_event]} =
+               conn
+               |> get("/v2/accounts/#{account}/activities",
+                 limit: 1,
+                 scope: "gen:#{height}-#{height}"
+               )
+               |> json_response(200)
+
+      assert %{
+               "height" => ^height,
+               "type" => "InternalTransferEvent",
+               "payload" => %{"kind" => "reward_block", "ref_tx_hash" => nil}
```

Comment:
May it please check the other kinds too?

Reply:
Done, checking more kinds now, including hardforks